### DISCUSSION TITLE
Scripts for periodic backups of specified servers.

### DISCUSSION THREAD

**Discussion by BenVlodgi** [Tyrrrz/DiscordChatExporter#997] [Q&A]
I'd like to backup all the raw data (JSON) for some discord servers.

I would like to:
- Run this throttled to a reasonable degree so as to not have my account flagged.
- Store all shared assets in the same folder per server. 
- Have the scripts pick up from the last time I ran the backup.

I've used the GUI, and read up on the CLI, I know some of the necessary flags and options. But I wanted to save the effort if someone else already has written this up already.

**Comment by Tyrrrz** [comment]
See the scheudling docs: https://github.com/Tyrrrz/DiscordChatExporter/tree/master/.docs
